packages/bot-skeleton/src/utils/date-time-helper.js

```diff
@@ -29,3 +31,27 @@ export const durationToSecond = duration => {
     }
     return 0;
 };
+
+export const timeSince = timestamp => {
+    const now = new Date();
+    const second_past = (now.getTime() - timestamp) / 1000;
+
+    if (second_past < 60) {
+        return localize('{{time_past}}s ago', { time_past: parseInt(second_past) });
```

Comment:
Perhaps we could provide the translators better context by naming the placeholders after their unit? e.g. `second_past`

Reply:
yeah sure